scripts/src/java/org/oppia/android/scripts/RegexPatternValidationCheck.kt

```diff
@@ -0,0 +1,232 @@
+package org.oppia.android.scripts
+
+import java.io.File
+import java.io.FileInputStream
+import org.oppia.android.app.model.FilenameChecks
+import org.oppia.android.app.model.FilenameCheck
+import org.oppia.android.app.model.FileContentChecks
+import org.oppia.android.app.model.FileContentCheck
+import org.oppia.android.scripts.ScriptResultConstants
+
+class RegexPatternValidationCheck {
```

Comment:
All public members should have KDocs, including for scripts. Please add here & elsewhere.

Reply:
Okay, added.